recipes/opene57/all/conanfile.py

```diff
@@ -65,8 +67,13 @@ def layout(self):
     def requirements(self):
         if self.options.with_tools:
             self.requires("boost/1.83.0")
```

Comment:
```suggestion
            self.requires("boost/1.84.0")
```

We have Boost 1.84.0 available too https://conan.io/center/recipes/boost?version=1.84.0